pkg/acceptance/helpers/security_integration_client.go

```diff
@@ -0,0 +1,93 @@
+package helpers
+
+import (
+	"context"
+	"testing"
+
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk"
+	"github.com/stretchr/testify/require"
+)
+
+type SecurityIntegrationClient struct {
+	context *TestClientContext
+	ids     *IdsGenerator
+}
+
+func NewSecurityIntegrationClient(context *TestClientContext, idsGenerator *IdsGenerator) *SecurityIntegrationClient {
+	return &SecurityIntegrationClient{
+		context: context,
+		ids:     idsGenerator,
+	}
+}
+
+func (c *SecurityIntegrationClient) client() sdk.SecurityIntegrations {
+	return c.context.client.SecurityIntegrations
+}
+
+func (c *SecurityIntegrationClient) CreateSaml2(t *testing.T, id sdk.AccountObjectIdentifier) (*sdk.SecurityIntegration, func()) {
+	t.Helper()
+	// generated by `openssl req -x509 -new -newkey rsa:2048 -nodes -subj '/C=US/ST=California/L=San Francisco/O=Snowflake/CN=Snowflake' -out x509_key.pem -days 36500`
+	x509 := `MIIDpzCCAo+gAwIBAgIUfg15OPhCN6lOivWEUoprAY27/5EwDQYJKoZIhvcNAQEL
```

Comment:
If you already extracted it, let's leave it, but copy the cert generation from the test.

Reply:
Done - moved to a separate function.